nwb_conversion_tools/datainterfaces/axonadatainterface.py

```diff
@@ -0,0 +1,316 @@
+"""Authors: Steffen Buergers"""
+import os
+import re
+import datetime
+import contextlib
+import mmap
+import numpy as np
+
+import spikeextractors as se
+
```

Comment:
```suggestion
```
You're really close to the import style we use - first block being things that come with Python naturally, second block being manually installed one, and third block being local project imports. So `spikeextractors `and `pynwb` go together here.